### ISSUE TITLE
Rocket.chat Taskbar icon is pixelated under Windows 10 [3.2.2]

### ISSUE THREAD

**Issue by blacklord049** [RocketChat/Rocket.Chat.Electron#2033]
Hi,

Operating system: Windows 10 x64 19042.985
App Version: 3.2.2
Description: Rocket.chat Taskbar icon is pixelated under Windows 10 (when labels are enabled in Taskbar settings). 

![image](https://user-images.githubusercontent.com/16919700/118231565-a1a1f180-b48f-11eb-8d83-2ab79a1bb9ef.png)

Current Behavior: pixelated icon looks vintage
Expected Behavior: having the same icon as the pinned one or when taskbar labels are hidden

![image](https://user-images.githubusercontent.com/16919700/118231837-f2b1e580-b48f-11eb-97cf-958c173b83cd.png)

Thank you

**Comment by ankar84** [comment]
I don't have that bug
![image](https://user-images.githubusercontent.com/4023037/118251449-43d1d180-b4b0-11eb-8b89-14b6c7a59ee5.png)
![image](https://user-images.githubusercontent.com/4023037/118251459-48968580-b4b0-11eb-900e-86e60f036991.png)

**Comment by blacklord049** [comment]
If I disable labels, the icon is fine, but if labels are enabled then the icon becomes ugly.

**Comment by bkraul** [comment]
Is this being looked at? My screen resolution is 3840x2160 (HDPI)

**Comment by jeanfbrito** [comment]
Please test if the problem persists with our latest release. I will be closing the issue but if there is any problem yet, we will open it again.

**Comment by bkraul** [comment]
@jeanfbrito Problem persists with release 3.8.7 (latest).

![image](https://user-images.githubusercontent.com/14616851/183064663-8441c87e-f591-4bf2-9d9b-8a5cf223924d.png)

**Comment by jeanfbrito** [comment]
This happens because you are using a 4k resolution with scale tu increase the size? It should be a problem with Windows in high resolutions.

**Comment by bkraul** [comment]
Yes, it is using font scaling. But I have had this setting all along and the pixelation was not occurring with previous versions of the RC client.

![image](https://user-images.githubusercontent.com/14616851/183073835-f7ab0773-6ef9-415c-961b-9bd2485e8ea4.png)

**Comment by jeanfbrito** [comment]
What exactly are the previous versions? Could you test if without scale it will work normally?
We already have a big resolution icon on the project, but I cant understand why its not being used in your case.

**Comment by bkraul** [comment]
This is 3.2.0 "with" scaling still turned on (as shown in the previous settings screenshot). As you can see, the icon looks perfect on HDPI

![image](https://user-images.githubusercontent.com/14616851/183082920-b4f7d5c4-d5c4-43b0-abc2-9a9e5b5da256.png)

**Comment by codeitua** [comment]
I have the same problem with custom icons without any scaling on 1920x1080 display with Windows 10 and Windows 11.
Also, if I delete custom icons, stock icon is also blurry.
![image](https://user-images.githubusercontent.com/2102318/189351859-163b9728-19ef-4cef-b9a5-003cd7a8bdeb.png)
